content/en/event-driven_architecture.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Event-Driven Architecture
+status: Feedback Appreciated
+category: concept
+---
+## What it is
+Event-driven architecture is a software architecture that promotes the creation, processing, and consumption of events. An event is any change of state or other significant occurrence in hardware or software. This architecture creates the structure in which events can be properly routed from their source to the desired receivers.
```

Comment:
Not seeing any chance here. Maybe you didn't commit?

Reply:
You are right, I did not commit before push. It's Friday